djangoproject/static/js/djangoproject.js

```diff
@@ -22,3 +22,15 @@ document.querySelectorAll('.messages li .close').forEach(function (el) {
     this.parentElement.classList.add('fade-out');
   });
 });
+
+// Check all console tab inputs of the same type when one's label is clicked
+document.querySelectorAll('.console-block label').forEach(function (el) {
+  el.addEventListener('click', function (e) {
+    const input_id = e.currentTarget.getAttribute('for');
+    const selector = input_id.endsWith('unix') ? '.c-tab-unix' : '.c-tab-win';
```

Comment:
Not sure if it's worth the added complexity, but a regexp could be used to not have to hardcode `unix` and `win`. What do you think?

```suggestion
    const input_id = e.currentTarget.getAttribute('for');
    const input_id_regex = /^c-tab-\d+-(?<platform>[a-z]+)$/;
    const platform = input_id.match(input_id_regex).groups.platform;
    const selector = `.c-tab-${platform}`;
```

Reply:
It's a fair question. As you said, it adds complexity in the form of a small feature that didn't exist in the old code, and I'm not trying to add features in these PRs. However, I'm happy to make the change if you think it's useful, but I'd probably be a -0 at this point.

I may be in favor of a simpler implementation:

```javascript
    const input_id = e.currentTarget.getAttribute('for');
    const selector = '.c-tab-' + input_id.split('-').at(-1);
```

Maybe this way of doing it is a good balance: not too complex, but it will also support possible future platforms as long as they don't contain a dash.

What is your preference between the three implementation options?

Reply:
I've had some time to think about it over the weekend, and I think it's fine as it is. The extra complexity is not warranted.

Reply:
Ok thanks!